src/test/regress/specs/isolation_insert_select_vs_all.spec

```diff
@@ -0,0 +1,97 @@
+#
+# How we organize this isolation test spec, is explained at README.md file in this directory.
+#
+
+# create range distributed table to test behavior of INSERT/SELECT in concurrent operations
+setup
+{
+	SET citus.shard_replication_factor TO 1;
+	CREATE TABLE insert_select_append(id integer, data text);
+	SELECT create_distributed_table('insert_select_append', 'id', 'append');
```

Comment:
we are testing self inserts on single shard append distributed tables. Is this viable scenario ?